.gitignore

```diff
@@ -1030,3 +1030,14 @@ service/__pycache__/db.cpython-38.pyc
 tests/__pycache__/__init__.cpython-38.pyc
 tests/api/__pycache__/__init__.cpython-38.pyc
 tests/api/__pycache__/test_drone.cpython-38-pytest-7.2.0.pyc
+config/__pycache__/db_config.cpython-38.pyc
+config/__pycache__/db_config.cpython-38.pyc
+crud/__pycache__/__init__.cpython-38.pyc
+crud/__pycache__/crud_drone.cpython-38.pyc
+crud/__pycache__/crud_session.cpython-38.pyc
+crud/__pycache__/crud_station.cpython-38.pyc
```

Comment:
Where I stopped: FAILED: No config file 'alembic.ini' found, or file has no '[alembic]' section
It seems like you lost some files.

Please, create a new folder, clone this branch to a new folder and try to recreate all steps that you need to start the application (including empty db!!!) . All steps fix to README.MD file